lucille-core/src/main/java/com/kmwllc/lucille/connector/storageclient/GoogleStorageClient.java

```diff
@@ -117,6 +118,19 @@ protected InputStream getFileReferenceContentStream(FileReference fileReference,
     return Channels.newInputStream(readChannel);
   }
 
+  @Override
+  protected String getStartingDirectory(TraversalParams params) {
+    URI pathURI = params.getPathToStorageURI();
+    String startingDirectory = Objects.equals(pathURI.getPath(), "/") ? "" : pathURI.getPath();
+    if (startingDirectory.startsWith("/")) return startingDirectory.substring(1);
```

Comment:
from a coding convention standpoint, we avoid one-line conditionals without braces around the body (unless it's the ? : syntax)...